packages/core/src/storybook/components/related-components/descriptions/virtualized-list-description/virtualized-list-description.jsx

```diff
@@ -1,8 +1,8 @@
 import { useMemo } from "react";
 import { RelatedComponent } from "vibe-storybook-components";
 import VirtualizedList from "../../../../../components/VirtualizedList/VirtualizedList";
-import { generateItems } from "../../../../../components/VirtualizedList/__stories__/virtualizedList.stories.helpers";
-import styles from "./virtualizedList.module.scss";
+import { generateItems } from "../../../../../components/VirtualizedList/__stories__/VirtualizedList.stories.helpers";
+import styles from "./virtualized-list.module.scss";
```

Comment:
I guess a mistake?

Reply:
Kept the same convention as the other storybook related components but I'm all for moving them also in the usual convention, but in a different PR

Reply:
in other related we have scss with kebab case?

Reply:
Yes, all of them. I don't really like it but I think it's out of scope for now, let's talk about it